lightkurve/periodogram.py

```diff
@@ -371,6 +508,40 @@ def flatten(self, method='logmedian', filter_width=0.01, return_trend=False):
             return snr, bkg
         return snr
 
+    def _get_fwhm(self, numax):
+        """For a given numax, the method calculates the expected Full Width Half
+        Maximum of the seismic mode envelope as (Mosser et al 2010) for RGB
+        stars:
```

Comment:
As a seismology-n00b, it's not clear to me exactly what a `seismic mode envelope` is. and why a specific `numax` has a predictable FWHM of the envelope.  Is this something that is easy to elucidate by adding one or two extra sentences?  If so, it would help @barentsen understand what's going on =)

Reply:
Added background + references